Defaults for xfig. Idea from Roland Rosenfeld <roland@debian.org>

--- xfig-3.2.9/app-defaults/Fig.orig	2023-08-15 11:11:22.000000000 +0200
+++ xfig-3.2.9/app-defaults/Fig	2023-10-29 12:13:38.520817101 +0100
@@ -12,6 +12,10 @@
 !
 !-----------------------------------------------------------------------
 
+! If the following resource is set to false, xfig will use metric units
+! by default.
+Fig.inches:     false
+
 ! The following fixes a problem that some window managers have
 ! with xfig continually resizing itself
 Fig*AllowShellResize:		False
